graphql-server/src/schemas/neo4j.ts

```diff
@@ -65,14 +50,25 @@ export async function buildNeo4jDbSchema() {
     }
   `;
 
-  const ogm = new OGM({ typeDefs: extendedTypeDefs, driver: neo4jDriver });
+  const features = {
+    filters: {
+      String: {
+        MATCHES: true,
+      },
+    },
+  };
```

Comment:
FYI I removed `ID: true` inside of `String: { ... }`. That config would let us perform RegEx matching on values of type `ID` ([source](https://neo4j.com/docs/graphql/current/queries-aggregations/filtering/)), which is a use case that we don't have.

Reply:
Isn't ID something we were interested in accessing?

Reply:
Yes, we are interested in accessing the ID, but setting `ID: true` here does something else. The docs link I shared above says it enables RegEx matching for value of type ID. For example, if we wanted to search for internal ID with containing value "abc". Maybe I missed something there?